zerver/tests/tests.py

```diff
@@ -1548,6 +1548,24 @@ def check_for_toggle_param(self, pattern, param):
         user_profile = get_user_profile_by_email("hamlet@zulip.com")
         self.assertEqual(getattr(user_profile, param), False)
 
+    def check_for_toggle_param_patch(self, pattern, param):
+        # type: (str, str) -> None
```

Comment:
Not needed, see https://chat.zulip.org/#narrow/near/102480/stream/GCI.20tasks/topic/testing.20toggle.20params